src/main/java/io/opentracing/contrib/jdbc/JdbcTracingUtils.java

```diff
@@ -33,6 +34,13 @@
    */
   static final StringTag PEER_ADDRESS = new StringTag("peer.address");
 
+  static final BooleanTag SLOW = new BooleanTag("slow");
+
+  /**
+   * can be modified by application code
+   */
+  public static int slowQueryThresholdMs = Integer.getInteger("io.opentracing.contrib.jdbc.slowQueryThresholdMs", 100);
```

Comment:
For backwards compatibility, I would use `0` as the default value, and skip bypass the new logic if slowQueryThresholdMs. This way people who want the feature can enable it with non-zero value, while others won't be paying the perf penalty of getting two more timers, etc.

Reply:
Updated, WDYT @malafeev ?